oauth_client.go

```diff
@@ -134,6 +135,9 @@ type OAuthClientCreateOptions struct {
 
 	// Required: The VCS provider being connected with.
 	ServiceProvider *ServiceProviderType `jsonapi:"attr,service-provider"`
+
+	// True if the oauth client is organization scoped, false otherwise.
+	OrganizationScoped *bool `jsonapi:"attr,organization-scoped,omitempty"`
```

Comment:
Nit, but in keeping with the other attributes I would add `Optional:` to the beginning of the comment. Also, it might be beneficial to describe what "organization scoped" means -- something like:

```suggestion
  // **Note: This field is still in BETA and subject to change.**
	// Optional: Whether the OAuthClient is available to all workspaces in the organization.
```